src/tests/ftest/soak/soak.py

```diff
@@ -114,7 +113,12 @@ def setUp(self):
         # self.output dir is an avocado directory .../data/
         self.log_dir = get_log_file("soak")
         self.outputsoakdir = self.outputdir + "/soak"
-        # Fail if slurm partition daos_client is not defined
+        # Create the remote log directories on all client nodes
+        self.test_log_dir = self.log_dir + "/pass" + str(self.loop)
+        self.local_pass_dir = self.outputsoakdir + "/pass" + str(self.loop)
+        self.sharedlog_dir = self.tmp + "/soak"
+        self.sharedsoakdir = self.sharedlog_dir + "/pass" + str(self.loop)
```

Comment:
If there was a need to make another change, I'd suggest the following:
```suggestion
        self.shared_soak_dir = os.path.join(self.shared_log_dir, "pass{}".format(self.loop))
```